portafly/src/i18n/i18n.tsx

```diff
@@ -13,6 +13,7 @@ const sections: Array<ITranslationsPages> = ['shared', 'overview', 'analytics',
 
 const options = {
   lng: EN,
+  localeExtension: 'yml',
```

Comment:
Why passing this option? Is not a valid option according to https://www.i18next.com/overview/configuration-options

Reply:
You are right, not needed